README.md

```diff
@@ -290,9 +297,9 @@ Caching, Importing Blocks, and Block Information
 
 </p></details>
 
-### Contributing to documentation for Open Ethereum packages
+### Contributing to documentation for OpenEthereum packages
```

Comment:
I think this whole paragraph can be deleted. What's the point in saying "document rust code using the rust documentation best practices"?